duckduckgo/macos-browser
# Update find-in-page to latest version

Task/Issue URL: https://app.asana.com/0/1177771139624306/1202241075893770/f
Tech Design URL:
CC:

**Description**:

Fixes find in page causing page breakage on some websites
Change: https://github.com/more-duckduckgo-org/duckduckgo-find-in-page/pull/12



**Steps to test this PR**:
1. Go to https://www.wpbeginner.com/best-woocommerce-hosting/
1. Open find-in-page (e.g. use Cmd+f)
1. Search for "best"
    - **Without fix:** Content shifts right
    - **With fix:** Content does not shift, as expected

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
